### PR TITLE
chore: 📝 updated II integration docs

### PR DESCRIPTION
**Pull request by letmejustputthishere**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [x] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [x] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: docs/developer-docs/integrations/internet-identity/integrate-identity.md [hunk 1]
```diff
@@ -104,4 +134,4 @@ Figuring the canister IDs, and using the `canisterId=...` query parameter is all
 - `npm run proxy`: Start a proxy that serves Internet Identity on `localhost:8086` and the webapp on `localhost:8087` for easy access.
 - `npm run test`: Start the proxy and run browser tests against the `internet_identity` canister.
 
-For more information, check the [`dfx.json`](https://github.com/dfinity/internet-identity/blob/main/demos/using-dev-build/dfx.json) file and the [Genesis talk on Internet Identity](https://youtu.be/oxEr8UzGeBo).
\ No newline at end of file
+For more information, check the [`dfx.json`](https://github.com/dfinity/internet-identity/blob/main/demos/using-dev-build/dfx.json) file, the [Genesis talk on Internet Identity](https://youtu.be/oxEr8UzGeBo) and the [SDK documentation](https://smartcontracts.org/docs/quickstart/quickstart-intro.html). Not bored yet? Check out the [Internet Computer Specification](https://smartcontracts.org/docs/introduction/welcome.html) and the [Internet Identity Specification](../../../references/ii-spec.md).
```
Comment (domwoe) [inline docs/developer-docs/integrations/internet-identity/integrate-identity.md:137]: Link to spec is broken and should be a relative link as well.

File: docs/developer-docs/integrations/internet-identity/integrate-identity.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+# Integrating with Internet Identity
 
-This shows how to integrate and test a project with Internet Identity. This uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
+This guide shows how to integrate and test a project with Internet Identity. The complete code example can be found [here](https://github.com/dfinity/internet-identity/tree/main/demos/using-dev-build). It uses the development [build flavor](https://github.com/dfinity/internet-identity/blob/main/README.md#build-features-and-flavors) of Internet Identity and the [agent-js](https://github.com/dfinity/agent-js) library.
 
 This is a standalone project that you can copy to your own project.
 
+## Prerequisites
+
+* [dfx](https://internetcomputer.org/docs/current/developer-docs/build/install-upgrade-remove)
```
Comment (sesi200) [inline docs/developer-docs/integrations/internet-identity/integrate-identity.md:9]: relative links, please

File: docs/developer-docs/integrations/internet-identity/integrate-identity.md [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+* Node.js v16+
+
+This tutorial assumes that you are already familiar with the [basic concepts of the IC](https://internetcomputer.org/docs/current/developer-docs/ic-overview) (canisters, how to use `dfx`, etc.).
```
Comment (sesi200) [inline docs/developer-docs/integrations/internet-identity/integrate-identity.md:12]: same here

File: docs/developer-docs/integrations/internet-identity/integrate-identity.md [hunk 4]
```diff
@@ -104,4 +134,4 @@ Figuring the canister IDs, and using the `canisterId=...` query parameter is all
 - `npm run proxy`: Start a proxy that serves Internet Identity on `localhost:8086` and the webapp on `localhost:8087` for easy access.
 - `npm run test`: Start the proxy and run browser tests against the `internet_identity` canister.
 
-For more information, check the [`dfx.json`](https://github.com/dfinity/internet-identity/blob/main/demos/using-dev-build/dfx.json) file and the [Genesis talk on Internet Identity](https://youtu.be/oxEr8UzGeBo).
\ No newline at end of file
+For more information, check the [`dfx.json`](https://github.com/dfinity/internet-identity/blob/main/demos/using-dev-build/dfx.json) file, the [Genesis talk on Internet Identity](https://youtu.be/oxEr8UzGeBo) and the [SDK documentation](https://smartcontracts.org/docs/quickstart/quickstart-intro.html). Not bored yet? Check out the [Internet Computer Specification](../../../references/ic-interface-spec.md) and the [Internet Identity Specification](../../../references/ii-spec.md).
```
Comment (sesi200) [inline docs/developer-docs/integrations/internet-identity/integrate-identity.md:137]: same here

**Comment by jessiemongeon1** [comment]
This PR appears to be outdated/not needed. Closing this PR to clean up the repo. Please feel free to reopen if you think this should be revisited.

**Pull request closed without merging**